docs/rnn.rst

```diff
@@ -168,8 +168,12 @@ Iterate (or not)
 ----------------
 
 The ``apply`` method of a recurrent brick accepts an ``iterate`` argument,
-which defaults to ``True``. Setting it to ``False`` causes the ``apply`` method
-to compute only one step in the sequence.
+which defaults to ``True``. It is the reason for passing a 3-dimensional tensor
```

Comment:
Yep, the input vector must normally have 1 dimension more that `apply` functions expects, but it does not have to be 3-dimensional.